android/src/main/java/com/reactnativecommunity/imageeditor/ImageEditorModule.java

```diff
@@ -173,6 +173,7 @@ public void cropImage(
       Promise promise) {
     ReadableMap offset = options.hasKey("offset") ? options.getMap("offset") : null;
     ReadableMap size = options.hasKey("size") ? options.getMap("size") : null;
+    boolean useInternalCache = options.hasKey("useInternalCache") ? options.getBoolean("useInternalCache") : true;
```

Comment:
It should default to false to match the current behaviour on https://github.com/callstack/react-native-image-editor
```suggestion
    boolean useInternalCache = options.hasKey("useInternalCache") ? options.getBoolean("useInternalCache") : false;
```

Reply:
Fixed :)